File: BasicOperations.c
#include<stdio.h>

int main()
{
    int a,b;
    printf("Enter 2 numbers : \n");
    scanf("%d %d",&a,&b);
    printf("Addition : %d",a+b);
    printf("\nSubtraction : %d",a-b);
    printf("\nMultiplication : %d",a*b);
    printf("\nDivision : %f",(float)a/b);
    return 0;
}

File: BasicOperations.java
import java.util.*;

class BasicOperations
{
        public static void main(String x[])
        {
            int a,b;
            Scanner obj1=new Scanner(System.in);
            System.out.println("Enter 2 numbers : ");
            a=obj1.nextInt();
            b=obj1.nextInt();
            System.out.println("Addition : "+(a+b));
            System.out.println("Subtraction : "+(a-b));
            System.out.println("Multiplication : "+(a*b));
            System.out.println("Division : "+(a/b));
        }
}
